### DISCUSSION TITLE
New blob emojis or selecting which ones you want.

### DISCUSSION THREAD

**Discussion by Kore-Development** [NamelessMC/Nameless#2150] [Ideas]
**Please describe the feature in as much detail as possible**
I have made issue #1396 about updating to EmojiOne 4.5 but apparently, there are problems with licenses. I did find out though that EmojiOne 2.2.7 has the same license as 3.0.0 (not in 3.0.1 and newer). Would I be possible to use 3.0.0 instead?

[EmojiOne 3.0.0 License](https://github.com/emojione/emojione/blob/v3.0.0/LICENSE.md)
[Example URL: https://cdn.jsdelivr.net/gh/emojione/emojione-assets@3.0.0/png/128/1f600.png](https://cdn.jsdelivr.net/gh/emojione/emojione-assets@3.0.0/png/128/1f600.png)

**Comment by samerton** [comment]
To be honest I'm not sure if this is intentional; the [readme](https://github.com/emojione/emojione/tree/v3.0.0#emojione-version-3) for 3.0.0 is set up to suggest the licensing is different.

For this reason I'll likely stick with 2.2.x just to be sure, but I can investigate making it easier to change emoji packs if you want to change it per site.

**Comment by Derkades** [comment]
The readme links to this page: https://www.emojione.com/licenses . Non-profits with <250 employees require the premium license. The free license is only for personal and educational use

**Comment by Derkades** [comment]
Actually, NamelessMC might qualify as "Personal Use". EmojiOne defines it as:

>* A project where no revenue is generated
>* A project for pure enjoyment or education
>* A project with zero connection to a business
>* Exception to the above: For internal testing purposes

NamelessMC is not a registered business and is not made with the intention of making money. Though, it does generate revenue through donations and is made not just for enjoyment, but with the intention of it being used in production.

It's a gray area, I personally wouldn't risk it

**Comment by samerton** [comment]
Unfortunately Nameless wouldn’t qualify for the free license. A possible alternative is Twemoji but more support for switching emoji packs would be required in Nameless.

**Comment by Kore-Development** [comment]
I also found some blob emojis. For example unicode 1F600 is [https://cdn.jsdelivr.net/gh/C1710/blobmoji@master/png/128/emoji_u1f600.png](https://cdn.jsdelivr.net/gh/C1710/blobmoji@master/png/128/emoji_u1f600.png) or [https://cdn.jsdelivr.net/gh/C1710/blobmoji@master/svg/emoji_u1f600.svg](https://cdn.jsdelivr.net/gh/C1710/blobmoji@master/svg/emoji_u1f600.svg):
![](https://cdn.jsdelivr.net/gh/C1710/blobmoji@master/png/128/emoji_u1f600.png)
[Github repository](https://github.com/C1710/blobmoji)
